pay_android/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1.0 (2023-02-02)
+
+* Bump `flutter_svg` to latest release.
```

Comment:
Suggest more specifity:
* Bump `flutter_svg` to version `2.0.5`

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3